recipes/ddtrace/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "ddtrace" %}
+{% set version = "0.42.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ddtrace-{{ version }}.tar.gz
+  sha256: dba66c7013a4352dfa20ed31e989001250056c89811d1cb5ee10e23dc98bac0a
+
+build:
+  number: 0
+  skip: true   # [py==30 or py==31 or py==32 or py==33 or py==34]
+  entry_points:
+    - ddtrace-run = ddtrace.commands.ddtrace_run:main
+    - pyddprofile = ddtrace.profiling.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython
+    - pip
+    - python
+    - setuptools_scm >=4
+  run:
+    - enum34  # [py<34]
+    - funcsigs >=1.0.0  # [py==27]
+    - intervaltree
+    - protobuf >=3
+    - python
+    - tenacity >=5
+
+test:
+  imports:
+    - ddtrace
+    - ddtrace.bootstrap
+  requires:
+    - pip
+
+about:
+  home: https://github.com/DataDog/dd-trace-py
+  summary: Datadog tracing code
+  license: BSD-3-Clause
+  license_file: LICENSE
```

Comment:
Please include the actual BSD license too

Reply:
Just added it - thank you for catching that